Render commands into toml before saving them.

```
[ERROR] 2022/09/19 10:00:07 Using Config File: /config/notifiarr.conf: getting config: config file: unmarshaling file [/config/notifiarr.conf]: toml: line 326 (last key "watch_file"): expected a top-level item to end with a newline, comment, or EOF, but got 'M' instead
```


Origin user: Davo1624
Origin message: [Discord Server Link](https://discord.com/channels/764440599066574859/764441658485112853/1021420549085335584)
Reporting user: TwitchCaptain
Reporting message: [Discord Server Link](https://discord.com/channels/764440599066574859/764441658485112853/1021475773795815514)

--- a/pkg/configfile/template.go
+++ b/pkg/configfile/template.go
@@ -3,6 +3,7 @@ package configfile
 import (
 	"fmt"
 	"runtime"
+	"strings"
 	"text/template"
 	"time"
 
@@ -23,6 +24,9 @@ func Funcs() template.FuncMap {
 	return map[string]interface{}{
 		"os":    func() string { return runtime.GOOS },
 		"force": func() bool { return ForceAllTmpl },
+		"toml": func(v string) string {
+			return strings.ReplaceAll(v, "'''", `''`)
+		},
 		"version": func() string {
 			return fmt.Sprintf("v%-7s @ %s", version.Version, time.Now().UTC().Format("060201T1504"))
 		},
@@ -533,7 +537,7 @@ bus_ids  = [{{range $s := .Snapshot.Nvidia.BusIDs}}"{{$s}}",{{end}}]
 
 [[command]]
   name    = '{{$item.Name}}'
-  command = '''{{$item.Command}}'''
+  command = '''{{toml $item.Command}}'''
   shell   = {{$item.Shell}}
   log     = {{$item.Log}}
   notify  = {{$item.Notify}}

